WebClient/Content/WindTemperatureView.css

```diff
@@ -0,0 +1,3 @@
+.d3-control-windTemperatureView {
```

Comment:
I think we should use the same naming convension evrywhere.
Mostly we use Lower Camel Case style therefore we can stick to it.
WindTemperatureView.css -> windTemperatureView.css

Reply:
Done